pages/dkp/kommander/2.2/workspaces/applications/appdeployments/index.md

```diff
@@ -0,0 +1,91 @@
+navigationTitle: AppDeployments
+title: AppDeployments
+excerpt: Use AppDeployments to deploy, and customize platform, DKP catalog, and custom applications in your environment
+menuWeight: 10
+beta: false
+enterprise: false
+---
+
+An `AppDeployment` is a [Custom Resource][CRD] created by DKP with the purpose of deploying applications (platform, DKP catalog and custom applications) in the management cluster, managed clusters, or both. Customers of both Essential and Enterprise products use `AppDeployments`, regardless of their setup (networked, air-gapped, etc.), and their infrastructure provider.
+
+When installing DKP, an `AppDeployment` resource is created for each enabled **platform application**. This `AppDeployment` resource references a `ClusterApp`, which then references the repository that contains a concrete declarative and preconfigured setup of an application, usually in the form of a [`HelmRelease`][helm]. `ClusterApps` are cluster-scoped so that these platform applications are deployable to all workspaces or projects.
+
+In the case of **DKP catalog** and **custom applications**, the `AppDeployment` references an `App` instead of a `ClusterApp`, which also references the repository containing the installation and deployment information. `Apps` are namespace-scoped and are meant to only be deployable to the workspace or project in which they have been created.
+
+For example, this is the default `AppDeployment` of the Kube Prometheus Stack platform application:
+
+```yaml
+apiVersion: apps.kommander.d2iq.io/v1alpha2
+kind: AppDeployment
+metadata:
+  name: kube-prometheus-stack
+  namespace: <your-workspace-namespace>
+spec:
+  appRef:
+    name: kube-prometheus-stack-33.1.5
+    kind: ClusterApp
+```
+
+## Customization
+
+If you want to customize an application, or change how a specific app is deployed, you can create a `ConfigMap` to change or add values to the information that is stored in the `HelmRelease`. Override the default configuration of an application by setting the `configOverrides` field on the `AppDeployment` to that `ConfigMap`.
+
+This is an example, of how to customize the `AppDeployment` of Kube Prometheus Stack:
+
+1.  Provide the name of a `ConfigMap` with the custom configuration in the `AppDeployment`:
+
+    ```yaml
+    cat <<EOF | kubectl apply -f -
+    apiVersion: apps.kommander.d2iq.io/v1alpha2
+    kind: AppDeployment
+    metadata:
+      name: kube-prometheus-stack
+      namespace: <your-workspace-namespace>
+    spec:
+      appRef:
+        name: kube-prometheus-stack-33.1.5
+        kind: ClusterApp
+      configOverrides:
+        name: kube-prometheus-stack-overrides-attached
+    EOF
+    ```
+
+1.  Create the `ConfigMap` with the name provided in the previous step, which provides the custom configuration on top of the default configuration:
+
+    ```yaml
+    cat <<EOF | kubectl apply -f -
+    apiVersion: v1
+    kind: ConfigMap
+    metadata:
+      namespace: ${WORKSPACE_NAMESPACE}
```

Comment:
If we want to use this envvar `${WORKSPACE_NAMESPACE}` we should have a step at the top setting it to the workspace namespace, and then reference it wherever we have `<your-workspace-namespace>`. I think it would be easier for the user to use the envvar e.g. like we do under prereqs on https://docs.d2iq.com/dkp/kommander/2.2/workspaces/applications/platform-applications/application-deployment/#prerequisites